packages/filetree/main/FileTreeChannel.js

```diff
@@ -1,4 +1,5 @@
 const { IpcChannel } = require('@obsidians/ipc')
+const { sortFile } = require('../../workspace/src/ProjectManager/helper')
```

Comment:
should look like this:
```suggestion
const { sortFile } = require('@obsidians/workspace')
```